### PR TITLE
oudated services versions

### PR DESCRIPTION
**Pull request by calvix**
script `fetch-yochu-assets` has old values for service versions, so i updated them to match the ones in `/etc/yochu-env` on vms
@giantswarm/team-positive

### CODE REVIEW

File: scripts/fetch-yochu-assets [hunk 1]
```diff
@@ -1,11 +1,11 @@
 #!/bin/sh
 
-ETCD_VERSION=v2.2.5-gs-1
+ETCD_VERSION=v3.0.3
 FLEET_VERSION=v0.11.5-gs-grpc-1
 DOCKER_VERSION=1.10.3
-YOCHU_VERSION=0.19.1
-RKT_VERSION=v1.3.0
-K8S_VERSION=v1.2.2
+YOCHU_VERSION=0.20.1
```
Comment (hectorj2f) [inline scripts/fetch-yochu-assets]: Do we really have yochu `0.20.1` ? https://github.com/giantswarm/yochu/blob/master/VERSION

**Comment by xh3b4sd** [comment]
The CI is not happy with your changes. Please have a look what went wrong with the versions.

**Comment by calvix** [comment]
@xh3b4sd I don't think that this related  to my changes, since I only changed values in fetching script, it has nothing to do with building the repository. 

in details build fails at downloading some public repos.

File: scripts/fetch-yochu-assets (hunk 1 above)
Comment (calvix) [inline scripts/fetch-yochu-assets]: this is what I got on newly installed machine and yes it works
```
core@000006d356aaabe5 ~ $ cat /etc/yochu-env 
YOCHU_VERSION=0.20.1
DOCKER_VERSION=1.10.3
ETCD_VERSION=v3.0.3
FLEET_VERSION=v0.11.5-gs-grpc-1
RKT_VERSION=v1.10.1
K8S_VERSION=v1.2.4
core@000006d356aaabe5 ~ $
```
fetch scripts can download yochu 0.20.1,
https://downloads.giantswarm.io/yochu/0.20.1/yochu

Comment (teemow) [inline scripts/fetch-yochu-assets]: There is no release of yochu 0.20.1. @hectorj2f according to the date it looks like it contains your iptables template changes. can you create the release?

Comment (hectorj2f) [inline scripts/fetch-yochu-assets]: Omg. My changes.... I can create it ofc

**Comment by hectorj2f** [comment]
LGTM, we have to fix the problems of the golang dependencies. Maybe we have to upgrade the golang dependencies.

**Pull request closed without merging**